<?xml version="1.0" ?><root>
<report><version>2</version><sessionid>dc9d616cb8ccfb65a1efe80</sessionid><type>desync</type><sku>ea.maxis.sims4.15.mac</sku><createtime>2017-11-30 02:23:47</createtime><buildsignature>Local.Unknown.Unknown.1.36.104.1220-1.200.000.311.Release</buildsignature><categoryid>scheduling.py:631</categoryid><desyncid>dc9d616cb8ccfb65a1efe80</desyncid><systemconfig/><screenshot/><desyncdata>Exception while processing the Master Controller. (HardStopError: Attempting to stop active handle to element &lt;SubclassableGen; _run_gen@52; running&gt; &lt;Gen; None; not-started&gt;)&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 195, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 203, in c_api_server_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\zone.py", line 434, in update&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\time_service.py", line 112, in update&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 217, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\elements.py", line 385, in _resume&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\master_controller.py", line 74, in _run_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\master_controller.py", line 568, in _process&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\sim.py", line 3231, in get_next_work&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\interaction_queue.py", line 944, in _get_head&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\interaction_queue.py", line 104, in get_next_unblocked_interaction_cancel_incompatible&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 2326, in get_sims_with_invalid_paths&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 2290, in _generate_connectivity&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\transition_sequence.py", line 1438, in compute_transition_connectivity&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\transition_sequence.py", line 1459, in run_transitions&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\transition_sequence.py", line 1238, in reset_derailed_transitions&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\sim.py", line 1334, in validate_current_location_or_fgl&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\base_object.py", line 266, in reset&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\services\reset_and_delete_service.py", line 298, in trigger_reset&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\services\reset_and_delete_service.py", line 416, in start_processing&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\services\reset_and_delete_service.py", line 440, in _process&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\services\reset_and_delete_service.py", line 538, in _process_one_record&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\elements.py", line 87, in trigger_hard_stop&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 554, in hard_stop&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 631, in _stop_element_tree&#13;&#10;scheduling.HardStopError: Attempting to stop active handle to element &lt;SubclassableGen; _run_gen@52; running&gt; &lt;Gen; None; not-started&gt;&#13;&#10;</desyncdata></report>
<report><version>2</version><sessionid>dc9d616cb8ccfb65a1efe80</sessionid><type>desync</type><sku>ea.maxis.sims4.15.mac</sku><createtime>2017-11-30 03:09:46</createtime><buildsignature>Local.Unknown.Unknown.1.36.104.1220-1.200.000.311.Release</buildsignature><categoryid>asm.py:985</categoryid><desyncid>dc9d616cb8ccfb65a1efe80</desyncid><systemconfig/><screenshot/><desyncdata>Exception in Sim Timeline: Exception running Element (RuntimeError: ASM :: Book_StartStop (HANDLE :: 390650)&gt;: Attempt to traverse between two states (entry -&gt; exit) where no valid path exists! Actors {'book': (&lt;weakref at 0x17256faf8; to 'book_Fun_Set4' at 0x182899b50&gt;, None, &lt;AnimationParticipant.TARGET = 103&gt;), 'x': (&lt;weakref at 0x160beb628; to 'object_sim' at 0x1541f99d0&gt;, None, &lt;AnimationParticipant.ACTOR = 101&gt;)})&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 212, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\elements.py", line 376, in _run&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\animation\animation_element.py", line 734, in do_begin&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\animation\asm.py", line 985, in request&#13;&#10;RuntimeError: ASM :: Book_StartStop (HANDLE :: 390650)&gt;: Attempt to traverse between two states (entry -&gt; exit) where no valid path exists! Actors {'book': (&lt;weakref at 0x17256faf8; to 'book_Fun_Set4' at 0x182899b50&gt;, None, &lt;AnimationParticipant.TARGET = 103&gt;), 'x': (&lt;weakref at 0x160beb628; to 'object_sim' at 0x1541f99d0&gt;, None, &lt;AnimationParticipant.ACTOR = 101&gt;)}&#13;&#10;</desyncdata></report>
</root>